core/src/main/scala/org/bitcoins/core/crypto/ExtKey.scala

```diff
@@ -203,8 +203,8 @@ sealed abstract class ExtPrivateKey
     val tweak = if (Secp256k1Context.isEnabled) {
       NativeSecp256k1.privKeyTweakAdd(il.toArray, key.bytes.toArray)
     } else {
-      val sum = BouncyCastleUtil.addNumbers(key.bytes, il)
-      sum.toByteArray
+      val sum = key.fieldElement.add(FieldElement(il))
+      sum.bytes.toArray
     }
     val childKey = ECPrivateKey(ByteVector(tweak))
```

Comment:
If `Secp256k1Context` is disabled you are doing `ByteVector` -> `Array[Byte]` -> `ByteVector`, would be better to have `tweak` just be a `ByteVector` and convert the ` NativeSecp256k1.privKeyTweakAdd` return to a `ByteVector`